sdc/tests/test_series.py

```diff
@@ -1257,6 +1257,23 @@ def test_impl(A):
         S = pd.Series([2, 4, 6], ['1', '3', '5'])
         np.testing.assert_array_equal(hpat_func(S), test_impl(S))
 
+    @skip_parallel
+    @skip_sdc_jit('Not impl in old style')
+    def test_series_loc_callable(self):
+        def test_impl(S):
+            return S.loc[(lambda a: a)]
+        hpat_func = self.jit(test_impl)
+        S = pd.Series([0, 6, 4, 7, 8], [0, 6, 66, 6, 8])
+        pd.testing.assert_series_equal(hpat_func(S), test_impl(S))
+
+    @unittest.skip('Loc callable return float Series')
+    def test_series_loc_callable2(self):
```

Comment:
What do you think about merging of `test_series_loc_callable` and `test_series_loc_callable2`?

Reply:
Why not do that?

Reply:
Because implementation has limit and always return float Series. In that case pandas return int Series, and we return float Series. I wrote about it in notes in loc. 
Test skips with expected failure